scripts/sas_ou_gpo_indicators.sh

```
@@ -0,0 +1,592 @@
+  value="${value%"${value##*[![:space:]]}"}"
+  printf '%s' "$value"
+}
+
+json_escape() {
+  printf '%s' "$1" \
+    | sed 's/\\/\\\\/g; s/"/\\"/g; s/\t/\\t/g; s/\r/\\r/g; s/\n/\\n/g'
+}
+
+csv_escape() {
+  local value="$1"
+  value="${value//"/""}"
```

Shellcheck (0.11.0)

[warning] Did you forget to close this double quoted string?

(SC1078)